saleor/graphql/utils.py

```diff
@@ -139,11 +139,15 @@ def filter_by_period(queryset, period, field_name):
 
 
 def generate_query_argument_description(search_fields):
+    deprecated_info = (
+        "DEPRECATED: Will be removed in Saleor 2.10,"
+        ' use "filter: {search: {}}" instead.\n'
```

Comment:
Playground can render fields nicely if you wrap them with backticks (`) so:
```suggestion
        " use `filter: {search: {}}` instead.\n"
```